modules/serverless-installing-cli-web-console.adoc

```diff
@@ -23,6 +23,8 @@ $ kn: No such file or directory
 ----
 ====
 
+* Optional. If you want to use the verification steps for this procedure, you must install the OpenShift (`oc`) CLI.
```

Comment:
Based on the [ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps) on optional and conditional steps, I think that this would fall under a conditional prereq. But I can see how this is certainly an edge case between optional and conditional. :) 

My reading on that guidance is that `Optional:` is to be used in procedure steps. In the OpenShift docs, we also use it in callouts. With those two points in mind, I would recommend cutting the Optional prefix.

If you do decide to leave it, the ISG does specify use of a colon over a period.